frontend/packages/kubevirt-plugin/integration-tests/tests/vm.guest.agent.scenario.ts

```diff
@@ -31,67 +30,77 @@ import {
+      vm_linux = new VirtualMachine(testVM.metadata);
+      createResource(testVM);
+      await vm_linux.detailViewAction(VM_ACTION.Start, false);
+      await vm_linux.waitForStatus(VM_STATUS.Running, SEC * 600);
+      await browser.sleep(GUEST_AGENT_FIELD_TIMEOUT_SECS);
+      await vm_linux.detailViewAction(VM_ACTION.Restart, false);
+      await vm_linux.waitForStatus(VM_STATUS.Running, SEC * 600);
+      execSync(`expect ${environmentExpecScriptPath} ${VM_LINUX_NAME} ${vm_linux.namespace} ${VM_LINUX_NAME}`);
+      await vm_linux.navigateToOverview();
+
+      //create windows VM
+       // for cmd-line scripts only
     execSync(`kubectl config set-context --current --namespace=${testName}`);
+    
     execSync('kubectl create -f -', {
       input: getFakeWindowsVM({
-        name: VM_NAME,
+        name: VM_WINDOWS_NAME,
         networkName: multusNAD.metadata.name,
-        vmIP: VM_IP,
+        vmIP: VM_WINDOWS_IP,
       }),
     });
-    vm = new VirtualMachine({ name: VM_NAME, namespace: testName });
 
-    // Wait for guest agent to load
-    await vm.navigateToDetail();
-    await browser.wait(
-      waitForStringInElement(vmView.vmDetailTimeZone(testName, VM_NAME), 'UTC'),
-      VM_WITH_GA_CREATE_AND_EDIT_CLOUDINIT_TIMEOUT_SECS,
-    );
+    vm_windows = new VirtualMachine({ name: VM_WINDOWS_NAME, namespace: testName });
+
   }, VM_WITH_GA_CREATE_AND_EDIT_CLOUDINIT_TIMEOUT_SECS);
 
   afterAll(async () => {
     deleteResource(multusNAD);
-    deleteResource(vm.asResource());
+    deleteResource(vm_linux.asResource());
+    deleteResource(vm_windows.asResource());
     execSync(`kubectl config set-context --current --namespace=default`);
   });
 
-  describe('Testing guest agent data', () => {
-    it('Displays guest agent data in Details tab', async () => {
-      expect(vmView.vmDetailHostname(testName, VM_NAME).getText()).toContain(VM_NAME);
-      expect(vmView.vmDetailTimeZone(testName, VM_NAME).getText()).toEqual('UTC');
-
-      await browser.wait(
-        until.presenceOf(vmView.vmDetailActiveUsersListNoUsers),
-        GUEST_AGENT_FIELD_TIMEOUT_SECS,
-      );
-
-      expect(vmView.vmDetailActiveUsersListNoUsers.getText()).toEqual('No Active Users');
-    });
+  const environmentExpecScriptPath = `${KUBEVIRT_SCRIPTS_PATH}/guest-agent-login.sh`;
+  const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 3000 * 60 * 5;
 
-    it('Displays guest agent data in Overview tab', async () => {
-      await vm.navigateToOverview();
-      expect(dashboardView.vmDetailsHostname.getText()).toContain(VM_NAME);
-      expect(dashboardView.vmDetailsTZ.getText()).toContain('UTC');
-      expect(dashboardView.vmDetailsLoggedInUsers.getText()).toEqual('No users logged in');
+  describe('Testing guest agent data', () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_EXTENDED_TIMEOUT_INTERVAL;
+    it('create new vm and install guest agent, login and verify the loggedin users expose', async () => {
```

Comment:
I mean keep the original tests and move your tests into the original tests, not to remove the original tests.

Reply:
ok, but this is the same result, doesn't it?

Reply:
The original tests examine datas on two tabs: overview and details, why move it into one? And the description is not precise if doing so.

Reply:
I've added the deleted test and re-organized all